cookie.go

```diff
@@ -26,6 +26,20 @@ func NewCookie(reporter Reporter, value *http.Cookie) *Cookie {
 	return newCookie(newChainWithDefaults("Cookie()", reporter), value)
 }
 
+// NewCookieC returns a new Cookie instance with config
+//
+// config and value should not be nil.
+//
+// Example:
+// cookie := NewCookieC(conf,&http.Cookie{...})
```

Comment:
Here and in other examples:

* let's rename `conf` to `config`, like in other places in documentation
* don't forget spaces between arguments: `conf,&http.Cookie{...}` -> `conf, &http.Cookie{...}`